pkgs/by-name/em/em100/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  curl,
+  fetchgit,
+  lib,
+  libusb1,
+  pkg-config,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "em100";
+  version = "unstable-2024-11-14";
+
+  src = fetchgit {
+    url = "https://review.coreboot.org/em100";
+    rev = "a78b4ba4774f05ecd7af495604b437113596d70e";
+    hash = "sha256-jzP56SMMiWiOynW17CFksi1VhpGt4oYYJrf4Rp9Vfs4=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    curl
+    libusb1
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make em100
```

Comment:
This breaks e.g. parallel building (which you should probably also enable) and other tweaks we have in the regular buildPhase.

Please use `buildFlags` instead.